catalog/gantt-chart/src/index.ts

```diff
@@ -222,21 +222,23 @@ Spotfire.initialize(async (mod) => {
                 color = node.rows()[0].color().hexCode;
             }
 
-            if (isCategorical) {
-                const colorValue = node.rows()[0].categorical("Color").value()[0].key;
-                const differentElements = node.rows().filter((r) => {
-                    return r.categorical("Color").value()[0].key !== colorValue;
-                }).length;
-                if (differentElements > 0) {
-                    color = config.defaultBarColor;
-                }
-            } else {
-                const colorValue = node.rows()[0].continuous("Color").value();
-                const differentElements = node.rows().filter((r) => {
-                    return r.continuous("Color").value() !== colorValue;
-                }).length;
-                if (differentElements > 0) {
-                    color = config.defaultBarColor;
+            if (hasColorExpression) {
+                if (isCategorical) {
+                    const colorValue = node.rows()[0].categorical("Color").value()[0].key;
```

Comment:
is this variable name correct? Isn't it a `colorValueKey`?

Reply:
This is however old code.